docs/tutorials/data_provider.md

```diff
@@ -133,11 +133,9 @@ where
 }
 
 // Usage example:
-let provider = icu_testdata::buffer();
-let lru_capacity = 100usize.try_into().unwrap();
```

Comment:
Nit: I gave `lru_capacity` its own name because I felt that it was more clear what the 100 was for. Please restore `lru_capacity`

Reply:
Issue: The only reason currently to use a cache in the ICU4X Data Provider is to cache the output of a buffer provider.